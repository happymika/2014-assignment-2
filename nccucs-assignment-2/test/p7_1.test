# p7 -- Simple test 1.

# Create a graph
CreateGraph Graph_test

# Add nodes
CreateNode n1 10
CreateNode n2 15
CreateNode n3 20
CreateNode n4 3
CreateNode n5 15
AddNode Graph_test n1
AddNode Graph_test n2
AddNode Graph_test n3
AddNode Graph_test n4
AddNode Graph_test n5

# Add edges
AddEdge Graph_test n1 n2
AddEdge Graph_test n2 n3
AddEdge Graph_test n2 n4
AddEdge Graph_test n3 n5
AddEdge Graph_test n4 n5

# Print some stuff
ListNodes Graph_test
ListChildren Graph_test n1
ListChildren Graph_test n2
ListChildren Graph_test n4
FindPath Graph_test n1 -> n5