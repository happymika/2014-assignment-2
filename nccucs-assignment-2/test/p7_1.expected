#p7 -- Simple test 1.

# Create a graph
created graph Graph_test

# Add nodes
created node n1 with cost 10
created node n2 with cost 15
created node n3 with cost 20
created node n4 with cost 3
created node n5 with cost 15
added node n1 to Graph_test
added node n2 to Graph_test
added node n3 to Graph_test
added node n4 to Graph_test
added node n5 to Graph_test

# Add edges
added edge from n1 to n2 in Graph_test
added edge from n2 to n3 in Graph_test
added edge from n2 to n4 in Graph_test
added edge from n3 to n5 in Graph_test
added edge from n4 to n5 in Graph_test

# Print some stuff
Graph_test contains: n1 n2 n3 n4 n5
the children of n1 in Graph_test are: n2
the children of n2 in Graph_test are: n3 n4
the children of n4 in Graph_test are:
shortest path in Graph_test: n1 n2 n4 n5